instrumentation/aws-sdk/aws-sdk-1.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v1_11/TracingRequestHandler.java

```diff
@@ -45,8 +45,10 @@
   public void beforeRequest(Request<?> request) {
     tracingHandler.beforeRequest(request);
     Context context = AwsSdkTracing.getOpenTelemetryContext(request);
-    Scope scope = context.makeCurrent();
-    request.addHandlerContext(SCOPE, scope);
+    if (context != null) {
```

Comment:
Do you think we can create a unit test for this? I guess we just need to initiate an AWS SDK call while another CLIENT span is active

Reply:
Precisely. That was the scenario that broke my testing and lead to this "if". I'll happily add a test for that.

Reply:
Added :)